kimchi/js/bindings.js

```diff
@@ -2068,9 +2071,20 @@ var caml_plonk_verifier_index_of_rust = function (x, affine_class) {
   // TODO: Handle linearization correctly!
   // var linearization = linearization_of_rust(x.linearization, affine_class);
 
+  function poly_comm_of_rust(poly_comm) {
+    return caml_poly_comm_of_rust_poly_comm(poly_comm, affine_class, false);
+  }
+  function lookup_verifier_index_of_rust(wasm_lookup_index) {
+    return caml_lookup_verifier_index_of_rust(
+      wasm_lookup_index,
+      poly_comm_class,
+      poly_comm_of_rust
+    );
+  }
+
   var caml_lookup_index = caml_opt_of_rust(
     x.lookup_index,
-    caml_lookup_verifier_index_of_rust
+    lookup_verifier_index_of_rust
```

Comment:
this was passing `caml_lookup_verifier_index_of_rust` a function which expects multiple arguments (value and conversion helpers), to `caml_opt_of_rust` which call it with only the value argument